libs/ui/src/components/NeoButton/NeoButton.scss

```diff
@@ -114,12 +114,15 @@
     }
   }
 
-  &--unlockable {
-    height: 40px;
+  &--secondary{
+    height: 22px;
+    line-height: 100%;
+    font-size: 12px;
```

Comment:
px?

Reply:
not my changes.
they come form a merge
check out this commit:

https://github.com/kodadot/nft-gallery/commit/0943d2ea573f1427895f5e98b85e8fff290b5702#diff-6cac81a09ae16a212a724a6359cce50c6031ffb32b5746b2fc6c34c2164e93e0

from this PR:

* https://github.com/kodadot/nft-gallery/pull/6199

Reply:
oh ☹️ sorry about that